vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -422,6 +419,36 @@ public VimeoAccount authorizeWithClientCredentialsGrantSync() {
         return call;
     }
 
+    /**
+     * Synchronous version of {@link #singleSignOnTokenExchange(String, AuthCallback)}. This is useful
+     * when already running in a background thread, such as when using the Android AccountManager.
+     *
+     * @param token           the authenticated user access token from application A. This <b>cannot</b> be a client
+     *                        credentials access token.
+     * @param accountName     the name of the account, usually the email
+     * @param basicAuthHeader a "basic" auth header to pass in; the basic auth header should be from
```

Comment:
I'm unclear as to why we need to pass in a basic auth header. Can you describe how this is used and where it comes from?